txscript/bench_test.go

```diff
@@ -224,6 +224,31 @@ func BenchmarkGetSigOpCount(b *testing.B) {
 	}
 }
 
+// BenchmarkGetPreciseSigOpCount  benchmarks how long it takes to count the
```

Comment:
Extra space after function name.

Reply:
Fixed locally along with the next case of it too.